orc8r/cloud/go/services/directoryd/servicers/servicer.go

```diff
@@ -160,3 +162,54 @@ func (d *directoryLookupServicer) GetNewSgwCTeid(ctx context.Context, req *proto
 	}
 	return &protos.GetNewSgwCTeidResponse{Teid: fmt.Sprint(sgwCTeid)}, nil
 }
+
```

Comment:
Note that if directoryd were more object oriented friendly, we could reduce the repetition of code. Maybe once we have time we can refactor all this and other functions which are just a duplication with different parameters